setup.py

```diff
@@ -132,12 +132,10 @@ def build_cmake(self, ext):
             '-DBUILD_PYTHON=ON',
             '-DCMAKE_POSITION_INDEPENDENT_CODE=TRUE',
             '-DBUILD_EXAMPLE=OFF',
-            '-DPYTHON_EXECUTABLE=' + sys.executable,
-            '-DPYTHON_LIBRARY=' + str(python_lib),
```

Comment:
Please remove the now unused code above (`pythonlib = ...`)

Reply:
Done